code-of-conduct.md

```diff
@@ -1,43 +1,64 @@
-# OpenContainers Code of Conduct
+# OpenContainers Code of Conduct v1.0
```

Comment:
I'd like to version things from now on so we can track it, just like any other software artifact 

I think this is close enough to ship TBH :)